docs/modules/ROOT/pages/getting-started/roles_persmissions.adoc

```diff
@@ -1,20 +1,20 @@
 = Permissions and roles
 
-As a workspace owner, you can https://placeholder.com[invite] one or more collaborators to your workspace by adding their user ID, such as an external SSO ID. After https://placeholder.com[adding a user to a workspace], you assign them a specific role that determines the actions they can take in a workspace.
+As a workspace owner, you can invite collaborators to your workspace by adding their user ID, such as an external SSO ID. After adding a user to a workspace, you assign them a specific role that determines the actions they can take in that workspace.
 
-The roles available for a workspace, from least access to most access, are:
+The roles available for a workspace, from least access to most access, are the following:
 
-* *Contributor*: Ideal for users who actively push to your workspace.
+* *Contributor*: For users who actively push to your workspace.
 
-* *Maintainer*: Ideal for users who need to manage the workspace without access to sensitive actions like managing security or deleting a workspace.
+* *Maintainer*: For users who need to manage the workspace without access to sensitive actions like managing security or deleting a workspace.
 
-* *Owner*: Ideal for users who need full access to the workspace, including sensitive and destructive actions like managing security or deleting a workspace.
+* *Owner*: For users who need full access to the workspace, including sensitive actions like managing security or deleting a workspace.
```

Comment:
It's not "Owner". This role is called "Admin". See https://github.com/redhat-appstudio/book/blob/main/ADR/0011-roles-and-permissions.md

Reply:
I just checked in the UI, and even though we call this role "admin" in the backend, the UI does call it "owner". Let's stick with what's in the UI - that's what the user will see.